expr/validator.go

```diff
@@ -9,6 +9,7 @@ import (
 var ErrIntPositive = errors.NewBadRequest("integer must be positive")
 var ErrInvalidAggFunc = errors.NewBadRequest("Invalid aggregation func")
 var ErrNonNegativePercent = errors.NewBadRequest("The requested percent is required to be greater than 0")
+var ErrWithinZeroOneInclusiveInterval = errors.NewBadRequest("The floating point must lie within interval [0,1]")
```

Comment:
```suggestion
var ErrWithinZeroOneInclusiveInterval = errors.NewBadRequest("value must lie within interval [0,1]")
```

This will ultimately create an error like "xFilesFactor: value must lie within interval [0,1]"

Reply:
Also, it seems like graphite-web doesn't validate. `<0` is just always true and `>1` is always false.